### PR TITLE
Update policy section

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js [hunk 1]
```diff
@@ -22,8 +22,6 @@ var operationModule = function () {
     var constants = require('/app/modules/constants.js');
     var devicemgtProps = require('/app/conf/devicemgt-props.js').config();
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:23]: Use proper camel case.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js [hunk 2]
```diff
@@ -75,13 +73,15 @@ var operationModule = function () {
...
+        var user = session.get(constants.USER_SESSION_KEY);
         var endPoint = devicemgtProps["httpsURL"] + '/' + deviceType + "/controller/" + operation;
         var header = '{"owner":"' + user.username + '","deviceId":"' + deviceId + '","protocol":"mqtt", "sessionId":"' + session.getId() + '"}';
-        log.info(params);
         return post(endPoint, params, JSON.parse(header), "json");
     };
 
     publicMethods.handleGETOperation = function (deviceType, operation, operationName, deviceId) {
+        var user = session.get(constants.USER_SESSION_KEY);
+        log.info("user: " + user);
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:84]: Is this log really necessary?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js [hunk 3]
```diff
@@ -31,6 +31,30 @@ policyModule = function () {
...
+        var carbonUser = session.get(constants["USER_SESSION_KEY"]);
+        if (!carbonUser) {
+            log.error("User object was not found in the session");
+            throw constants["ERRORS"]["USER_NOT_FOUND"];
+        }
+        if (policyName && deviceType) {
+            var queName = "wso2/iot/" + carbonUser.username + "/" + deviceType;
+            log.info("Queue : " + queName);
+
+            var mqttsenderClass = Packages.org.wso2.device.mgt.mqtt.policy.push.MqttPush;
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:44]: Use camel-case appropriately.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js [hunk 4]
```diff
@@ -31,6 +31,30 @@ policyModule = function () {
...
+            var mqttsender = new mqttsenderClass();
+
+            var policyPretext = "POLICY:";
+            var policyPayload = policyPretext + policyDefinition;
+
+            var result = mqttsender.pushToMQTT(queName, policyPayload, "tcp://192.168.67.21:1883", "Raspberry-Policy-sender");
+            log.info(result);
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:51]: Is this log necessary?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policy.create.wizard/wizard.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+function onRequest(context){
+    context.handlebars.registerHelper('equal', function (lvalue, rvalue, options) {
+        if (arguments.length < 3)
+            throw new Error("Handlebars Helper equal needs 2 parameters");
+        if( lvalue!=rvalue ) {
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policy.create.wizard/wizard.js:23]: Fix formatting issues.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.policy.listing/listing.js [hunk 6]
```diff
@@ -1,30 +1,28 @@
...
-        context["policyListToView"] = [];
-        context["policyListingStatusMsg"] = "An unexpected error occured @ backend. Please try again later.";
-        context["saveNewPrioritiesButtonEnabled"] = false;
-        context["noPolicy"] = true;
+        pageData["policyListToView"] = [];
+        pageData["saveNewPrioritiesButtonEnabled"] = false;
+        pageData["noPolicy"] = true;
     }
-    return context;
+    log.info(pageData);
```
Comment (harshanL) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.policy.listing/listing.js:26]: Is this log necessary?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js (hunk 1 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:23]: Is necessary to use camel case for file names?

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js (hunk 2 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js:84]: Fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js (hunk 3 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:44]: All variables are already in camel case.

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js (hunk 4 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js:51]: Fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.policy.listing/listing.js (hunk 6 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.policy.listing/listing.js:26]: Fixed in commit  https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

File: features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policy.create.wizard/wizard.js (hunk 5 above)
Comment (charithag) [inline features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policy.create.wizard/wizard.js:23]: Fixed with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1

**Pull request merged**